docs/getting-started/running-oracles.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Running Oracles
+description: How to run an oracle for Mento, the stability protocol.
+---
+
+# Running oracles
+
+Oracles are a fundamental piece for Mento, the stability protocol behind Celo stable assets. Their propose is to forward to the blockchain the price of CELO/USD, CELO/EUR and CELO/BRL. You can read the [Celo Oracle documentation](celo-codebase/protocol/stability/oracles) to get an overview of how they work.
+
+# Getting started
+
+Oracles work by running a client that fetches the price from centralized exchanges (CEX) and push them on-chain by calling `SortedOracles.report(address token, uint256 value, address lesserKey, address greaterKey)`. SortedOracles is a [Celo Core Contract](/community/release-process/smart-contracts).
+
+A [reference implementation](https://github.com/celo-org/celo-oracle) of such client is written in TypeScript and would be used for this guide. Releases for this client can be found [here](https://github.com/celo-org/celo-oracle/releases).
+
+## Requriments
+* One VM dedicated for each oracle is recommended, but it is acceptable that they run multiple instances in the case they are for different stables.
+* A dedicated full node running in its own VM. Minimal hardware requirements and instructiosn on how to run a full node can be found [here](/getting-started/mainnet/running-a-full-node-in-mainnet#:~:text=Full%20nodes%20play%20a%20special,other%20full%20nodes%20and%20validators.).
+* The private key of an address on Celo, which can be stored on a private key file, on a HMS or hosted in the full nodes itself. More information about each can be found below.
+
+It is not strictly required but it is recommended to have the [Celo CLI](/command-line-interface/introduction#what-is-the-celo-cli) avaiable at least in your local enviroment, and ideally in each VM. It could be specially useful to respond to on-call.
```

Comment:
```suggestion
It is not strictly required but it is recommended to have the [Celo CLI](/command-line-interface/introduction#what-is-the-celo-cli) available at least in your local environment, and ideally in each VM. It could be especially useful to respond to on-call.
```